web/src/search_suggestion.js

```diff
@@ -626,6 +626,14 @@ function get_sent_by_me_suggestions(last, operators) {
 }
 
 function get_operator_suggestions(last) {
+    // Offer suggestion for deprecated "is:private" filter
```

Comment:
Better would be "// Suggest is:dm to anyone with is:private in their finger memory", since that explains the purpose of this logic a bit more precisely.

Reply:
I used 'muscle memory' since that seemed like a more generic / known term.